public-api-prd/ingress.yaml

```diff
@@ -1,27 +1,27 @@
-# apiVersion: networking.k8s.io/v1
-# kind: Ingress
-# metadata:
-#   name: bd-dvp-api-ingress
-#   namespace: bd-dvp
-#   annotations:
-#     cert-manager.io/cluster-issuer: letsencrypt-production
-#     nginx.ingress.kubernetes.io/client-body-buffer-size: 1M
-#     nginx.ingress.kubernetes.io/global-rate-limit: '100'
-#     nginx.ingress.kubernetes.io/global-rate-limit-window: 1m
-# spec:
-#   ingressClassName: nginx
-#   tls:
-#     - hosts:
-#         - api.dev.osrsbotdetector.com
-#       secretName: api-tls
-#   rules:
-#     - host: api.dev.osrsbotdetector.com
-#       http:
-#         paths:
-#           - path: /
-#             pathType: Prefix
-#             backend:
-#               service:
-#                 name: bd-dvp-api-service
-#                 port:
-#                   name: dvp-api-port
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: pub-api-ingress
+  namespace: bd-dvp
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+    nginx.ingress.kubernetes.io/client-body-buffer-size: 1M
+    nginx.ingress.kubernetes.io/global-rate-limit: '100'
+    nginx.ingress.kubernetes.io/global-rate-limit-window: 1m
+spec:
+  ingressClassName: nginx
+  tls:
+    - hosts:
+      - api-v2.dev.osrsbotdetector.com # this is mainly for the purposes such that the /docs are available
+      secretName: api-tls

```

Comment:
this secret name is already in use (in the same namespace) by https://github.com/Bot-detector/bot-detector-k8s/blob/develop/bd-core-dvp/ingress.yaml#L16